### PR TITLE
feat: Add basic Github Actions workflow and clean up index

### PR DESCRIPTION
**Pull request by bnussman-akamai**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,7 +1,5 @@
 import { noCustomFontWeight } from './rules/no-custom-fontWeight';
 
-module.exports = {
-  rules: {
-    'no-custom-fontWeight': noCustomFontWeight,
-  },
+export const rules = {
+  'no-custom-fontWeight': noCustomFontWeight,
```
Comment (abailly-akamai) [inline src/index.ts:4]: I _think_ this should be fine but eslint is weird

File: .github/workflows/publish.yml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        with:
+          name: dist
+          path: dist/
+      - name: Publish to NPM
+        continue-on-error: true
+        env:
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+        run: |
+          npm config set //registry.npmjs.org/:_authToken ${NPM_TOKEN}
+          npm publish --access public
```
Comment (abailly-akamai) [inline .github/workflows/publish.yml:72]: BUN

**Review by abailly-akamai — APPROVED** [review]
Thanks!

**Comment by abailly-akamai** [comment]
@bnussman-akamai you also wanna bump the version?

File: src/index.ts (hunk 1 above)
Comment (bnussman-akamai) [reply src/index.ts:4]: I'll revert if it breaks stuff. I think `tsc` will transform this

**Comment by bnussman-akamai** [comment]
@abailly-akamai I'll bump after I merge this!

File: .github/workflows/publish.yml (hunk 2 above)
Comment (bnussman-akamai) [reply .github/workflows/publish.yml:72]: Should make the dependency install fast. We'll see how it goes

**Pull request merged**